package.json

```diff
@@ -66,6 +66,7 @@
     "webpack-cli": "^4.8.0"
   },
   "devDependencies": {
+    "@babel/plugin-transform-runtime": "^7.15.0",
```

Comment:
What is this doing ?   Not urgent for approval, just curious what impact this has on our project

Reply:
I needed this for babel to work with async/await.